RandomData.Api.Tests/NumberGenerators/NumberParametersValidatorTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using RandomData.Api.NumberGenerators.Dto;
+using RandomData.Api.NumberGenerators.Validator;
+using Xunit;
+
+namespace RandomData.Api.Tests.NumberGenerators
+{
+    public class NumberParametersValidatorTests
```

Comment:
I don't think that packages should be tested by us. I think they should be tested by package maintainer.

Reply:
I don't test packages, I test my rules - if they are defined correctly

Reply:
What I mean by testing packages is that you should assume that whenever you do something in package it'll work correctly. So if you create a rule that X must be lower than 10 you are sure that it will be correctly validated, because the tests to test whether it works are supposed to be in the package source files and thus should not be your concern.

I also might be wrong, so @krzysztof-gwozdz need to verify this ;)

Reply:
I still test my rules, if nothing is missing in sets defined by me. I understand what you say, I just don't agree :)